tgui/package.json

```diff
@@ -20,24 +20,16 @@
     "tgui:tsc": "tsc"
   },
   "dependencies": {
-    "@babel/core": "^7.15.0",
-    "@babel/eslint-parser": "^7.15.0",
-    "@babel/plugin-transform-class-properties": "^7.23.3",
-    "@babel/plugin-transform-jscript": "^7.14.5",
-    "@babel/preset-env": "^7.15.0",
-    "@babel/preset-react": "^7.23.3",
-    "@babel/preset-typescript": "^7.15.0",
+    "@swc/core": "^1.5.0",
```

Comment:
Of note, this seems to work here in this PR, but I get no candidates found for this version in the branch I'm testing.
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/d00dc4ef-5d16-453c-82ce-4ce5ffd7ac6c)

Reply:
Might be a yarn issue,
https://github.com/swc-project/swc/issues/8898